.ci/goreleaser/preremove.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# SPDX-FileCopyrightText: 2025 Siemens AG
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if command -v systemctl &> /dev/null; then
```

Comment:
Hm, this script – as the others – use `/bin/sh` as shell which could effectively be dash, bash, zsh, fish, .... and `command -v` also resolves shell functions: So if there's a shell function aptly named `systemctl` defined, e.g., in the global config file, that shell function is called instead of the `systemctl` executable on disk, which could be OK or not. On the other hand, in strictly POSIX shells, one cannot use `type -p` (which resolves in `$PATH` only) as parameters (as `-p`) to `type` are not POSIX defined/mandated. Well, there's no silver bullet here...